src/Reflection/Php/PhpMethodReflection.php

```diff
@@ -223,8 +222,8 @@ private function isVariadic(): bool
 			$filename = $this->declaringTrait->getFileName();
 		}
 
-		if (!$isNativelyVariadic && $filename !== null && is_file($filename)) {
```

Comment:
`filemtime` will return false when $filename is not a file. the code already handles this case.

lets drop this unnecessary file IO